File: 3 - OlaStoryboard/3 - OlaStoryboard/ViewController.m
//
//  ViewController.m
//  3 - OlaStoryboard
//
//  Created by André Guedes on 28/12/15.
//  Copyright © 2015 André Ricardo de Souza Guedes. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)fechar:(id)sender {
    [self dismissViewControllerAnimated:YES completion:nil];
}

@end

File: Course iOS9/Idade Felina/Idade Felina/ViewController.swift
//
//  ViewController.swift
//  Idade Felina
//
//  Created by André Ricardo de Souza Guedes on 12/07/16.
//  Copyright © 2016 André Ricardo de Souza Guedes. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var txtCatAge: UITextField!
    @IBOutlet weak var txtResult: UILabel!
    
    @IBAction func findAge(sender: AnyObject) {
        var catAge = Int(txtCatAge.text!)!
        catAge = catAge * 7
        txtResult.text = "A idade do seu gato é \(catAge)"
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

